src/components/Controls.vue

```diff
@@ -125,15 +125,15 @@ export default {
 
   methods: {
     startSimulation() {
-      this.$refs.nodelink.startSimulation();
+      this.$refs.MultiLink.startSimulation();
     },
 
     stopSimulation() {
-      this.$refs.nodelink.stopSimulation();
+      this.$refs.MultiLink.stopSimulation();
     },
 
     releaseNodes() {
-      this.$refs.nodelink.releaseNodes();
+      this.$refs.MultiLink.releaseNodes();
```

Comment:
The capitalization on these needs to be fixed.

Reply:
Fixed in a9ec392